### ISSUE TITLE
Migrate set up and deployment to tag-based approach

### ISSUE THREAD

**Issue by ebma** [pendulum-chain/pendulum-squids#78]
## Context
Subsquid introduced a new system to manage deployments. Instead of using version numbers and having a single production deployment, they want to use tags for each deployment. More info [here](https://docs.sqd.dev/deployments-two-release-notes/). 

Since the configuration we use currently, with designated production deployments, is deprecated and might be removed in the future, we should change our setup to support the tag based approach.

## TODO

- Adjust the parameters in each manifest file
- Change the instructions of the README to describe how to publish new versions to production
- Make sure that other integrations are aware of the new tag-based endpoints. These integrations are Zenlink and the Pendulum portal.

**Comment by ebma** [comment]
@pendulum-chain/product this is a nice-to-have feature that might become important sooner or later. It's unclear when subsquid will stop supporting our current setup so the squids might break in the future.

**Comment by prayagd** [comment]
IMO we should priortize this to avoid sudden breaking of things wdyt @pendulum-chain/devs ?

**Comment by prayagd** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Pendulum-6408a0365cda59c57d39bee5/issues/gh/pendulum-chain/pendulum-squids/78?planning-poker) with Zenhub @ebma @gianfra-t @Sharqiewicz

**Comment by ebma** [comment]
Yes, we should do it eventually but maybe we can sit it out for a few more weeks/month.

**Comment by prayagd** [comment]
I will move this to the bottom of the ready column, incase we have free bandwidth this could be picked up
